.gitignore

```diff
@@ -31,4 +31,4 @@ build/
 
 ### VS Code ###
 .vscode/
-/src/main/resources/application.properties
```

Comment:
For the password in the application. properties, we should not just add it in the .gitignore file since we still need the information stored in the file and if the file missing the application is unable to run for a user. We can encrypt it.  Source:https://martinfowler.com/bliki/TechnicalDebt.html